content/en/error_tracking/backend/exception_replay.md

```diff
@@ -21,88 +21,138 @@ aliases:
+|---|---|---|---|
+| **How to Enable** | Enabled by default | Settings page | Environment variables |
+| **Agent Version** | v7.49.0+ | v7.49.0+ | v7.49.0+ |
+| **Minimum Tracer Versions** | Python ≥ 3.15.0<br>Java ≥ 1.54.0<br>.NET ≥ 3.29.0<br>PHP: N/A | Python ≥ 3.10.0<br>Java ≥ 1.48.0<br>.NET ≥ 3.29.0<br>PHP: N/A	 | Python ≥ 1.16.0<br>Java ≥ 1.47.0<br>.NET ≥ 2.53.0<br>PHP ≥ 1.12.1 |
+| **Remote Configuration Required?** | Yes | Yes | No |
 
-Exception Replay is only available in APM Error Tracking. It is not available for errors sourced from Logs and RUM.
+To enable Exception Replay in-app, navigate to the Exception Replay **Settings** page in Error Tracking, select the
+desired environment or service, and toggle it to **Enabled**.
 
-## Setup
+{{< img src="tracing/error_tracking/error_tracking_exception_replay_enablement.mp4" video="true" alt="Enabling Exception Replay through the setting page" style="width:90%" >}}
 
-1. Upgrade the Datadog Agent to version `7.49.0` or higher.
-1. Upgrade the APM tracer library to the minimum required version or higher:
-   * `ddtrace` version `2.21.9+`
-   * `dd-trace-java` version `1.49.0+`
-   * `dd-trace-dotnet` version `3.18.0+`
-   * `dd-trace-php` version `1.12.1+`
-1. Run your service with the `DD_EXCEPTION_REPLAY_ENABLED` environment variable set to `true`.
-1. [Create a logs index][9] and configure it to the desired retention with no sampling.
-   * Set the filter to match on the `source:dd_debugger` tag.
-   * Ensure that the new index takes precedence over any others with filters that match that tag, because the first match wins.
+If in-app enablement isn't available, set the environment variable:
+
+```bash
+DD_EXCEPTION_REPLAY_ENABLED=true
+```
+
+This can also be used to override in-app configuration and takes precedence when both are set.
+
+### Create a logs index for Exception Replay snapshots
+
+Create a logs index dedicated to Exception Replay snapshots and configure it with the desired retention and no sampling.
+
+- Set the filter to match `source:dd_debugger`.
+- Ensure the index takes precedence over other indexes matching this tag (the first match wins).
 
 <div class="alert alert-info">
-<b>Why create a logs index?</b> When an error occurs and is captured in an APM span, Exception Replay variable snapshots are captured as logs with reference links to the APM span. When viewing the error in Error Tracking Explorer, variable snapshots from the log data display alongside stack trace details.
+<b>Why create a logs index?</b>
+Exception Replay snapshots are emitted as logs enriched with links back to the originating APM spans.
 </div>
 
-### Redacting sensitive data
+### Link your source code
+
+If you enable the Datadog Source Code Integration, you can see code previews directly inside your Error Tracking stack
+traces. When Exception Replay snapshots are captured, you can hover over variable names in the code preview to view
+their captured values.
```

Comment:
if you are in the mood of capture video, a short movie of the mouse hover would be very nice here.